portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!', 'And if you can\'t do it, if you aren\'t willing to keep looking for light in the darkest of places without stopping, even when it seem imposssible, you will never succeed'];
```

Comment:
For very long strings, it's usually easier to read if you break it into multiple lines with concatenation. Also you can choose to use single quote 'foo', or double quote "foo", which would determine which character needs to be escaped, so you can do something like,

"And if you can't do it, " +
"if you aren't willing to keep..." + ...

Reply:
I updated it. I just added foo = "..." or do i still need to break it up?

Reply:
I didn't meant to add foo. It is  to just show examples how you can choose to use single quote or double quote to define strings to avoid needing to escape special characters.

Breaking apart long expressions into multiple lines makes it easier to read, though I will leave it up to you to decide whether you want to do it